labs/gps_lab/README.md

```diff
@@ -0,0 +1,68 @@
+#GPS LAB
+
+##Description
+GPS Lab uses a Raspberry PI with a GPS sensor to fetch current location (latitude and longitude)
+ and sends the info to IoT hub.
+
+ A web application reads the data from IoT Hub real time and displays the GPS location in a Azure MAP.
+
+ ##Steps
+ ### Create and configure IoT Hub
+ 1. Create one azure IoT Hub and create one device under it 
+ 2. Install azure CLI (https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+ 3. Add a consumer group to your IoT hub (Using azure CLI OR from Azure Portal)
+       
+        az iot hub consumer-group create --hub-name YourIoTHubName --name YourConsumerGroupName
+ 4. Get a service connection string for your IoT hub
+        
+        az iot hub show-connection-string --hub-name YourIotHub --policy-name service
+ 5. Note down the consumer group and service connection string
+ 6. Get the connection string for the device you have created. Note that it will be different from the service connection string
+ 
+ Please refer to this document for detailed instructions (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-live-data-visualization-in-web-apps)
+
+
+ ###Client Side Code
+ 1. Set up and install the OS for Raspberry PI.
+ 2. Connect the pi with a monitor using micro HDMI or VNC connector
+ 3. Connect the GPS sensor to Raspberry PI
+       a. See **Raspberry Pi Pins.jpg** for reference 
+       b. Put the vcc to 2, ground to 6 tx and rx to 8 and 10
+       ![Image](./Client/Raspberry_Pi_Pins.jpg "Pins")
```

Comment:
Need a blank line to make the image appear on the line below, not inline with the text